container/20230206.md

```diff
@@ -0,0 +1,67 @@
+# 2023-02-06
+## Participants (github handles preferred)
+* @o-otte
+* @ajfriesen
+* @master-caster
+* @mxmxchere
+* @fkr
+* @garloff
+* @mafechner
+* @jnull
+* @matofeder
+* @mpilka
+* @joshmue
+* @frosty-geek
+* @jschoone
+
+## Community call report and minutes transfer
+  @jschoone
+
+## Agenda
+See Kanban board 
+
+### Ingress and Loadbalancing (issues/#227)
+* issues/#251: octavia ovn provider appears to be providing wanted L3 LB functionality
+- Available on gx-scs, pco, not on wavestack (ovn not used)
```

Comment:
Indentation is lost. Using the etherpad's export feature (as text file) should help a bit.

Reply:
@joshmue if you look at the branch, one sees the commit that I did last night to fix the formatting. No clue why it does not show up here.